docs/contents/components/forms/rating/accessibility.ja.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Rating
+description: "`Rating`は、ユーザーがアイテムに対して評価を行うためのUI要素で、星やハートなどのアイコンで提供され、ユーザーが選択した評価を視覚的に表示するコンポーネントです。"
+package-name: "@yamada-ui/rating"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+Ratingは、アクセシビリティに関して[WAI-ARIA - Radio Group Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/)に従います。
+
+## キーボード操作
+
+| キー                      | 説明                                                                                                                                          | 状態 |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---- |
+| `Tab`                     | 選択された項目をフォーカスします。項目が選択されていない場合は、最初の項目をフォーカスします。                                                | -    |
+| `ArrowLeft`, `ArrowUp`    | 前の項目をフォーカスして選択します。最初の項目の場合は、最後の項目をフォーカスして選択します。以前に選択されていた項目は、未選択にします。    | -    |
+| `ArrowRight`, `ArrowDown` | 次の項目をフォーカスして選択します。最後の項目の場合は、最初の項目をフォーカスして選択します。以前に選択されていた項目は、未選択にします。 　 | -    |
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性         | 使い方                                                   |
+| :------------- | :------------------- | :------------------------------------------------------- |
+| `Rating`       | `role="radiogroup"`  | ラジオグループであることを示します。                     |
+| `Rating`       | `aria-label`         | 選択されたアイテムの数を示します。                       |
+|                | `aria-readonly`      | `isReadOnly`が設定されている場合は`"true"`を設定します。 |
+|                | `aria-disabled`      | `isDisabled`が設定されている場合は`"true"`を設定します。 |
+| `RatingItem`   | `aria-label`         | `value`を設定します。                                    |
+| `RatingIcon`   | `aria-hidden="true"` | `RatingIcon`をアクセシビリティツリーから除外します。     |
```

Comment:
Add internal tag to `RatingIcon`.

https://yamada-ui.com/components/data-display/tag/accessibility